src/services/SyncService.ts

```diff
@@ -198,6 +201,13 @@ class SyncService {
 		}
 	}
 
+	_emitDocumentStateStep(documentState: string, version: number) {
```

Comment:
This is typescript so we could do `private emitDocumentStateStep` as far as I see we mostly used `_` as a convention for private methods